source/iface/command-accessor.rst

```diff
@@ -6,281 +6,274 @@
 
 .. _buffer-accessor:
 
-******************
+==================
 ``sycl::accessor``
```

Comment:
I need a table with targets from this part and a reference like ``_access-target`` to this table for the buffers chapter

Reply:
Added with https://github.com/imvucic/SYCL_Reference/pull/20/commits/7b6e03dc9cca91697627588f7f7cab406545983a